r_environment.yml

```diff
@@ -0,0 +1,27 @@
+name: alsf_r_analysis
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::jupyter
+  - conda-forge::jupyterlab
+  - conda-forge::ipykernel
+  - conda-forge::python
```

Comment:
if you are not using pip, you probably do not need these

Reply:
Thanks for catching that! I will remove those now!